README.md

```diff
@@ -6,9 +6,10 @@
 npm install minidump
 ```
 
-## Building
+## Building (for development)
 
-* `git clone --recurse-submodules https://github.com/electron/node-minidump`
+* `git clone https://github.com/electron/node-minidump`
+* `npm run submodule`
```

Comment:
I don't see how this is clearer than before. I'd prefer to keep the version on the left.

Reply:
Reverted